src/go/agwd/config/config.go

```diff
@@ -82,15 +110,19 @@ type ConfigManager struct {
 }
 
 func newDefaultConfig() *config.AgwD {
+
 	return &config.AgwD{
 		LogLevel:                        config.AgwD_INFO,
 		SctpdDownstreamServiceTarget:    "unix:///tmp/sctpd_downstream.sock",
 		SctpdUpstreamServiceTarget:      "unix:///tmp/sctpd_upstream.sock",
 		MmeSctpdDownstreamServiceTarget: "unix:///tmp/mme_sctpd_downstream.sock",
 		MmeSctpdUpstreamServiceTarget:   "unix:///tmp/mme_sctpd_upstream.sock",
 		// Sentry is disabled if DSN is not set.
-		SentryDsn:           "",
-		ConfigServiceTarget: "127.0.0.1:50090",
+		SentryDsn:               "",
+		VagrantPrivateNetworkIp: "192.168.60.142",
+		ConfigServicePort:       "6000",
+		CaptureServicePort:      "6001",
```

Comment:
in these cases do we want `net.Listen("tcp", ":" + port)`? perhaps we don't bother going through a `resolver.Target`

Reply:
probably cleaner - since i don't see us accessing these services through sockets, ill repurpose GetLocalHostTarget